test/e2e/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/migration/primarykey/NonePrimaryKeyMigrationE2EIT.java

```diff
@@ -0,0 +1,102 @@
+import org.apache.shardingsphere.test.e2e.data.pipeline.util.AutoIncrementKeyGenerateAlgorithm;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertFalse;
+
+@RunWith(Parameterized.class)
+@Slf4j
+public final class NonePrimaryKeyMigrationE2EIT extends AbstractMigrationE2EIT {
+    
+    private final PipelineTestParameter testParam;
+    
+    public NonePrimaryKeyMigrationE2EIT(final PipelineTestParameter testParam) {
+        super(testParam);
+        this.testParam = testParam;
+    }
+    
+    @Parameters(name = "{0}")
+    public static Collection<PipelineTestParameter> getTestParameters() {
+        Collection<PipelineTestParameter> result = new LinkedList<>();
+        if (PipelineBaseE2EIT.ENV.getItEnvType() == PipelineEnvTypeEnum.NONE) {
+            return result;
+        }
+        for (String version : PipelineBaseE2EIT.ENV.listStorageContainerImages(new MySQLDatabaseType())) {
+            result.add(new PipelineTestParameter(new MySQLDatabaseType(), version, "env/scenario/primary_key/none_primary_key/mysql.xml"));
+        }
+        return result;
+    }
+    
+    @Override
+    protected String getSourceTableOrderName() {
+        return "t_order";
+    }
+    
+    @Test
+    public void assertTextPrimaryMigrationSuccess() throws SQLException, InterruptedException {
+        log.info("assertTextPrimaryMigrationSuccess testParam:{}", testParam);
+        createSourceOrderTable();
+        try (Connection connection = getSourceDataSource().getConnection()) {
+            AutoIncrementKeyGenerateAlgorithm generateAlgorithm = new AutoIncrementKeyGenerateAlgorithm();
+            PipelineCaseHelper.batchInsertOrderRecordsWithGeneralColumns(connection, generateAlgorithm, getSourceTableOrderName(), PipelineBaseE2EIT.TABLE_INIT_ROW_COUNT);
+        }
+        addMigrationProcessConfig();
+        addMigrationSourceResource();
+        addMigrationTargetResource();
+        createTargetOrderTableRule();
+        startMigration(getSourceTableOrderName(), getTargetTableOrderName());
+        String jobId = listJobId().get(0);
+        waitIncrementTaskFinished(String.format("SHOW MIGRATION STATUS '%s'", jobId));
+        proxyExecuteWithLog("REFRESH TABLE METADATA", 1);
+        assertTargetAndSourceCountAreSame();
+        if (PipelineBaseE2EIT.ENV.getItEnvType() == PipelineEnvTypeEnum.DOCKER) {
+            commitMigrationByJobId(jobId);
+            List<String> lastJobIds = listJobId();
+            assertThat(lastJobIds.size(), is(0));
+        }
```

Comment:
We could just commit migration job, no matter env type. And also in TextPrimaryKeyMigrationE2EIT. Keep consistent with other migration E2E IT cases.